src/main/kotlin/mate/academy/Main.kt

```diff
@@ -1,5 +1,18 @@
 package mate.academy
 
+import mate.academy.service.FigureSupplier
+
+const val ARRAY_SIZE = 6
+
 fun main() {
-    // draw all figures here
+    val array = Array(ARRAY_SIZE) { FigureSupplier().getDefaultFigure() }
+    for (i in 0 until array.lastIndex) {
```

Comment:
You may improve current logic via setting until as array.size/2 this would allow avoid if statement in loop body.
Consider this in the future in similar situations